src/v2/Apps/Order/Components/__tests__/AddressModal.jest.tsx

```diff
@@ -29,6 +29,7 @@ const testAddressModalProps: Props = {
   address: savedAddress,
   onSuccess: jest.fn(),
   onError: jest.fn(),
+  onDeleteAddress: jest.fn(),
```

Comment:
I'd expect to see some tests in here to cover...

* the delete button appears at the right time
* clicking the delete button spawns a dialog
* when the dialog is confirmed, the "delete" action happens
* when the dialog is cancelled, the "delete" action doesn't happen

Are these intended to be added in a follow-up PR, or as part of this work?

Reply:
I think it's done now)